model/config.go

```diff
@@ -137,6 +137,9 @@ const (
 	SAML_SETTINGS_DEFAULT_NICKNAME_ATTRIBUTE   = ""
 	SAML_SETTINGS_DEFAULT_LOCALE_ATTRIBUTE     = ""
 	SAML_SETTINGS_DEFAULT_POSITION_ATTRIBUTE   = ""
+	SAML_SETTINGS_DEFAULT_SIGNATURE_ALGORITHM  = ""
```

Comment:
I think these should default to the current values, no?

Reply:
I went back and forth between current values and blank.  I went with blank mainly because our current values (SHA-1) are not good default values. We should probably default to (SHA-256).  However, as a new feature, I agree, the default should be the existing values.

As is written, the algorithms still exist in authnrequest.go and don't get overwritten if the config entry is missing or empty(""). So it actually works exactly as before.

```if s.SPSignatureAlgorithm != nil && *s.SPSignatureAlgorithm != "" {
		r.Signature.SignedInfo.SignatureMethod.Algorithm = *s.SPSignatureAlgorithm
	}

That said, I went back and forth myself, so I can set the defaults if so desired.
@enahum

Reply:
Yes I think administrators should see the current values (default ones) and not and empty field.